questions/qa-personal-names.fr.html

```diff
@@ -25,7 +25,7 @@
 // also make sure that the lang attribute on the html tag is correct!
 
 // TRANSLATORS should fill in these assignments:
-f.translators = '<a href="http://www.nota-bene.org/">Stéphane Deschamps</a> (Relecture par <a href="http://arkhi.org/">Fabien Basmaison</a> et Goulven Champenois)'; // translator(s) and their affiliation - a elements allowed, but use double quotes for attributes
+f.translators = '<a href="http://www.nota-bene.org/">Stéphane Deschamps<\/a> (Relecture par <a href="http://arkhi.org/">Fabien Basmaison<\/a> et Goulven Champenois)'; // translator(s) and their affiliation - a elements allowed, but use double quotes for attributes
```

Comment:
```suggestion
f.translators = '<a href="https://www.nota-bene.org/">Stéphane Deschamps<\/a>, <a href="https://arkhi.org/">Fabien Basmaison<\/a>  (Relecture par Fabien Basmaison et Goulven Champenois)'; // translator(s) and their affiliation - a elements allowed, but use double quotes for attributes
```

Reply:
If I understand what you’re doing here, shouldn’t your name also appear in the proof‑reader as well?

Reply:
I'm OK either way.